src/site/content/en/blog/streams/index.md

```diff
@@ -273,38 +273,38 @@ console.log(`The stream is ${locked ? 'indeed' : 'not'} locked.`);
 ### Readable stream code samples
 
 The code sample below shows all the steps in action. You first create a `ReadableStream` that in its
-`underlyingSource` argument defines a `start` method that tells the stream's `controller` to
+`underlyingSource` argument (that is, the `TimestampSource` class) defines a `start` method
```

Comment:
```suggestion
`underlyingSource` argument (that is, the `TimestampSource` class) defines a `start()` method
```

Reply:
I think this tries to convey too much in a single sentence, which I'm having trouble following. There doesn't have to be a 1 to 1 relationship between lines of text and lines of code, but it helps if single gestures in the code get at least their own suggestions.